proto-pruner/build.gradle

```diff
@@ -16,6 +16,7 @@ jar {
 dependencies {
   implementation project(':wire-schema')
   implementation deps.kotlin.stdlib.jdk8
+  implementation deps.guava
```

Comment:
is this dependency being pulled in from wire-schema?  if so, should it remain as an api dependency there?

Reply:
That's how it was before the change, but in fact both use Guava as implementation.